### PR TITLE
Update `UKCC11` tournament article with Qualifiers results and RO32 mappool

### PR DESCRIPTION
**Pull request by Bazzadwarf**
Updates for the end of Qualifiers and the beginning of the Round of 32.

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/UKCC/11/en.md [hunk 1]
```diff
@@ -47,8 +47,73 @@ The UK Community Cup 11 is run by various community members.
...
+| #29 | **Some Femboy Shit** | ![][flag_GB] [Kwichi-](http://osu.ppy.sh/users/16287466), ![][flag_GB] [Fluxey](http://osu.ppy.sh/users/9098192) |
+| #30 | **Platform Duty** | ![][flag_GB] [idealnettles](http://osu.ppy.sh/users/4010354), ![][flag_GB] [- Syndi -](http://osu.ppy.sh/users/11617315) |
+| #31 | **cheese gaming** | ![][flag_GB] [SinqHD](http://osu.ppy.sh/users/15887523), ![][flag_GB] [CrypticInsanity](http://osu.ppy.sh/users/11508900) |
+| #32 | **PopKO** | ![][flag_GB] [Claus](http://osu.ppy.sh/users/4966424), ![][flag_GB] [SpiralKO](http://osu.ppy.sh/users/5387803) |
+
 ## Mappools
 
+### Round of 32
+
+**[Download the mapack here (105 MB)](https://www.dropbox.com/s/yaoob4h4vmy8edy/UKCC%2011%20Round%20of%2032.zip?dl=0)**
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/11/en.md:93]:
```suggestion
**[Download the mappack here (105 MB)](https://www.dropbox.com/s/yaoob4h4vmy8edy/UKCC%2011%20Round%20of%2032.zip?dl=0)**
```

Comment (Bazzadwarf) [reply wiki/Tournaments/UKCC/11/en.md:93]: Of course the one gosh darn `Download the mappack here` line I copied had to be the only one with a mistake in it...

Comment (Walavouchey) [reply wiki/Tournaments/UKCC/11/en.md]: where did you copy it from? O.o

Comment (Bazzadwarf) [reply wiki/Tournaments/UKCC/11/en.md]: https://github.com/ppy/osu-wiki/blob/91765271b7f41e8b94aa595d48c89820430adaff/wiki/Tournaments/OWC/2021/en.md?plain=1#L248

Comment (Bazzadwarf) [reply wiki/Tournaments/UKCC/11/en.md]: Deserves me right for not using the template...

Comment (Walavouchey) [reply wiki/Tournaments/UKCC/11/en.md]: i guess it wasn't so bad, since you managed to find the *one* instance of this in the entire wiki

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**